### DISCUSSION TITLE
a problem in 4.2Example using Mars MOC images

### DISCUSSION THREAD

**Discussion by squidward-qwq** [NeoGeographyToolkit/StereoPipeline#430] [Q&A]
Hello, I have just started working with ASP, and my goal is to generate a DTM using ASP and stereo image pairs of Mars. However, I encountered some issues while generating the Mars DTM. First, I downloaded ASP and ISIS according to the UserGuide and successfully configured the environment.

Then, when conducting the experiment in section 4.2, I used ISIS to convert two test files, M0100115.imq and E0201461.imq, into .cub files and performed radiometric calibration. After that, I processed the stereo image pair using the command provided in the UserGuide. The generated DTM is shown in the image below. This image is not a grayscale image, and it hardly shows any topographic information.

Do you know what might be causing this issue? Is there a solution you could suggest? I would appreciate your guidance.
![Uploading QQ图片20240320205743.png…]()

**Comment by squidward-qwq** [comment]
sorry this is the picture
![QQ图片20240320205743](https://github.com/NeoGeographyToolkit/StereoPipeline/assets/88065797/847b2979-fe7a-48ed-9f28-cffea9b35ad9)

**Comment by oleg-alexandrov** [comment]
It looks that you only created the PC file. This is a point cloud and will not show up well. You need to create a DEM out of it. This is described  at https://stereopipeline.readthedocs.io/en/latest/next_steps.html#visualising. I also edited a bit the documentation in section 4.2 to make it more clear.

**Comment by squidward-qwq** [comment]
@oleg-alexandrov 
Thank you for your response; I successfully generated the DEM, which was of great help to my experiment. I will read the UserGuide carefully. Thank you very much.
